.github/workflows/verify_consumer_pacts.yaml

```diff
@@ -110,12 +110,33 @@ jobs:
     outputs:
       provider-version: ${{ steps.verification-test.outputs.provider-version }}
 
+    services:
```

Comment:
I don't like having to spin up a postgres in this step just for verifying consumer pacts. I am going to ticket up an alternative using either [TestContainers](https://testcontainers.com) or [zonky](https://github.com/zonkyio/embedded-database-spring-test).

Reply:
Talked offline; turns out I wasn't mocking the spring deps properly, so I don't think this is necessary. See this [commit](https://github.com/DataBiosphere/terra-landing-zone-service/pull/466/commits/675d0cbe1fd665ab4ff2bccffacfef0a3f099068).